src/mapping/RadialBasisFctSolver.h

```diff
@@ -0,0 +1,181 @@
+#include "impl/BasisFunctions.hpp"
+#include "precice/types.hpp"
+#include "utils/EigenHelperFunctions.hpp"
+#include "utils/Event.hpp"
+
+namespace precice {
+extern bool syncMode;
+
+struct RadialBasisFctSolver {
```

Comment:
Is this a class (with members having to preserve some invariance) or just a struct holding some information in a structured way?

A class would require accessors for `_matrixA` and `_qr`.
A struct wouldn't need `public:` and `private:`. 

```suggestion
class RadialBasisFctSolver {
```